ocpp/v201/enums.py

```diff
@@ -1364,6 +1387,11 @@ class UnitOfMeasureType(StrEnum):
     k = "K"
 
 
+UnitOfMeasureType = DeprecatedEnumWrapper(
```

Comment:
Not sure it worth the effort. Have these enums been part of any stable release.
Considering rc versions as API stable is unusual.

Reply:
i'm mixed on this one. 

correct, there has been no official stable release.

But... since 0.26.0 has been in the wild for the past year, there are definitely people who are using/depending on the library

I leaned towards usability for the community in this case but I could be convinced otherwise.

Reply:
@jerome-benoit actually, given that these OCPP 2.0 files are included in this library's 1.0 release (on pypi), I think it's better to be safe. I don't think the project has made it clear if the library's release version is tied to the OCPP version or not?